tests/components/ecobee/test_repairs.py

```diff
@@ -0,0 +1,82 @@
+"""Test repairs for Ecobee integration."""
+
+from http import HTTPStatus
+from unittest.mock import MagicMock
+
+from homeassistant.components.ecobee import DOMAIN
+from homeassistant.components.notify import DOMAIN as NOTIFY_DOMAIN
+from homeassistant.components.repairs import DOMAIN as REPAIRS_DOMAIN
+from homeassistant.components.repairs.issue_handler import (
+    async_process_repairs_platforms,
+)
+from homeassistant.components.repairs.websocket_api import (
+    RepairsFlowIndexView,
+    RepairsFlowResourceView,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from .common import setup_platform
+
+from tests.typing import ClientSessionGenerator, WebSocketGenerator
+
+THERMOSTAT_ID = 0
+
+
+async def test_ecobee_repair_flow(
+    hass: HomeAssistant,
+    mock_ecobee: MagicMock,
+    hass_client: ClientSessionGenerator,
+    hass_ws_client: WebSocketGenerator,
+) -> None:
+    """Test the ecobee notify service repair flow is triggered."""
+    assert await async_setup_component(hass, REPAIRS_DOMAIN, {REPAIRS_DOMAIN: {}})
```

Comment:
I think ecobee should have a dependency to repairs in the manifest, then this isn't needed anymore